### PR TITLE
Added initial content for ARCHER2 4cab resource limits.

### PR DESCRIPTION
**Pull request by mbareford**

### CODE REVIEW

File: user-guide/scheduler.rst [hunk 1]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
 Resource Limits
 ---------------
 
-There are different resource limits on ARCHER2 for different purposes.
+The ARCHER2 resource limits for any given job are covered by three separate attributes.
+
+* The amount of *primary resource* you require (i.e., CPU cores).
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:121]: On ARCHER2, the primary resource is "nodes", not "CPU cores"

File: user-guide/scheduler.rst [hunk 2]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+Each of these attributes is described in more detail below. 
+
+The *primary resources* you request are *compute* resources: CPU cores on the compute nodes. Other node resources,
+such as memory, are assigned pro rata based on the primary resource that you request.
+
+.. warning::
+
+   On ARCHER2, you cannot specify the memory for a job using the ``--mem`` options to Slurm
+   (e.g. ``--mem``, ``--mem-per-cpu``, ``--mem-per-gpu``). The amount of memory you are 
+   assigned is calculated from the amount of primary resource you request.
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:134]: As "--exclusive" is enforced on ARCHER2, there is no pro-rata calculation of memory - you get access to all of the memory per node, no matter what distribution of processes/threads you ask for.

File: user-guide/scheduler.rst [hunk 3]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+however, if you request 1 core, then you will be assigned a maximum of 256/128 = 2 GB of the memory available on the node
+(for high memory nodes the memory per core is 512/128 = 4 GB).
 
 .. note::
 
-   Details on the resource limits will be added when the ARCHER2 system
-   is available.
+   Using the ``--exclusive`` option in jobs will give you access to the full node memory even
+   if you do not explicitly request all of the CPU cores on the node.
+
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:149]: See above, this does not apply on ARCHER2

File: user-guide/scheduler.rst [hunk 4]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+.. list-table:: ARCHER2 Partitions
+   :widths: 30 50 20
+   :header-rows: 1
+
+   * - Partition
+     - Description
+     - Total nodes available
+   * - standard
+     - CPU nodes with AMD EPYC 7742 64-core processor :math:`\times` 2
+     - 992
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:171]: I think this should be "1024" even though they are not all up at the moment

File: user-guide/scheduler.rst [hunk 5]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+using the ``--partition`` option in your submission script. The following table has a list 
+of active partitions on ARCHER2.
+
+.. list-table:: ARCHER2 Partitions
+   :widths: 30 50 20
+   :header-rows: 1
+
+   * - Partition
+     - Description
+     - Total nodes available
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:168]: "Max. nodes available" is probably a more accurate label for this column

File: user-guide/scheduler.rst [hunk 6]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+   some is retained for running the operating system and other system processes.
+
+Partitions
+~~~~~~~~~~
+
+On ARCHER2, compute nodes are grouped into partitions. You will have to specify a partition
+using the ``--partition`` option in your submission script. The following table has a list 
+of active partitions on ARCHER2.
+
+.. list-table:: ARCHER2 Partitions
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:162]: Is there a "long" partition or is "long" just a QoS? Maybe just a QoS as it uses the same nodes as the "standard" partition...

File: user-guide/scheduler.rst [hunk 7]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+On ARCHER2, job limits are defined by the requested Quality of Service (QoS), as specified by the ``--qos`` Slurm directive.
+The  following table lists the active QoS on ARCHER2.
+
+.. list-table:: ARCHER2 QoS
+   :header-rows: 1
+
+   * - QoS
+     - Max Nodes
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:189]: I think this should be "Max. nodes per job". Or have I misunderstood?

File: user-guide/scheduler.rst [hunk 8]
```diff
@@ -116,15 +116,117 @@ will cancel (if waiting) or stop (if running) the job with ID ``12345``.
...
+
+:: 
+
+  sacctmgr show assoc user=$USER cluster=archer2 format=cluster,account,user,qos%50
+
+
+.. note::
 
-.. TODO: Add in partition and QOS limits once they are known
+  If you have needs which do not fit within the current QoS, please contact the Service Desk and we can discuss how to accommodate your requirements.
```
Comment (aturner-epcc) [inline user-guide/scheduler.rst:227]: Make "contact the Service Desk" a link to the Service Desk page on the ARCHER2 website: https://www.archer2.ac.uk/support-access/servicedesk.html

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Just a few minor comments to address

File: user-guide/scheduler.rst (hunk 1 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 2 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 3 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 5 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 6 above)
Comment (mbareford) [reply user-guide/scheduler.rst:147]: There's just a standard partition and three QoS (standard, short and long).

File: user-guide/scheduler.rst (hunk 8 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 7 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

File: user-guide/scheduler.rst (hunk 4 above)
Comment (mbareford) [reply user-guide/scheduler.rst]: OK

**Review by aturner-epcc — APPROVED** [review]
Thanks @mbareford LGTM

**Pull request merged**